components-core/src/main/java/com/adyen/checkout/components/core/paymentmethod/PaymentMethodDetails.kt

```diff
@@ -65,7 +65,12 @@ abstract class PaymentMethodDetails : ModelObject() {
                 DotpayPaymentMethod.PAYMENT_METHOD_TYPE -> DotpayPaymentMethod.SERIALIZER
                 EPSPaymentMethod.PAYMENT_METHOD_TYPE -> EPSPaymentMethod.SERIALIZER
                 EntercashPaymentMethod.PAYMENT_METHOD_TYPE -> EntercashPaymentMethod.SERIALIZER
-                GiftCardPaymentMethod.PAYMENT_METHOD_TYPE -> GiftCardPaymentMethod.SERIALIZER
+                GiftCardPaymentMethod.PAYMENT_METHOD_TYPE,
+                PaymentMethodTypes.MEAL_VOUCHER_FR_GROUPEUP,
+                PaymentMethodTypes.MEAL_VOUCHER_FR_NATIXIS,
+                PaymentMethodTypes.MEAL_VOUCHER_FR_SODEXO,
+                PaymentMethodTypes.MEAL_VOUCHER_FR -> GiftCardPaymentMethod.SERIALIZER
```

Comment:
Should we create a separate PaymentMethod object for meal vouchers, in case we need to change it from the `GiftCardPaymentMethod` later? (which would cause a breaking change)

Reply:
For now we will go on with the current solution, then consider refactoring the flow.